doc/configcommands.dsv

```diff
@@ -73,7 +73,8 @@ oldreader-flag-star|<flag>|""|If this is set and The Old Reader support is used,
 oldreader-login|<login>|""|This variable sets your The Old Reader login for The Older Reader support.|oldreader-login "your-login"
 oldreader-min-items|<number>|20|This variable sets the number of articles that are loaded from The Old Reader per feed.|oldreader-min-items 100
 oldreader-password|<password>|""|This variable sets your The Old Reader password for The Old Reader support. Double quotes should be escaped, i.e. you should write +{backslash}"+ instead of +"+.|oldreader-password "here_goesAquote:{backslash}""
-oldreader-passwordfile|<path-to-file|""|A more secure alternative to the above, by storing your password elsewhere in your system.|oldreader-passwordfile "path-to-file"
+oldreader-passwordfile|<path-to-file>|""|A more secure alternative to the above, by storing your password elsewhere in your system.|oldreader-passwordfile "path-to-file"
+oldreader-passwordeval|<command>|""|An even more secure alternative, is storing your password encrypted and decrypting it with an external command. For example with gpg or your systems keyring.|oldreader-passwordeval "command some-parameter"
```

Comment:
Technically this isn't inherently any more secure than `oldreader-password`, because user can do ``oldreader-passwordeval `echo mysecret` ``. So I'd just say it's "another option that lets you get your password from GPG, a keyring or any other source", or something like that.